Engine/platform/windows/setup/winsetup.cpp

```diff
@@ -428,9 +428,9 @@ class WinSetupDialog
 public:
     enum GfxModeSpecial
     {
-        kGfxMode_None    = -1,
-        kGfxMode_Desktop = -2,
-        kGfxMode_GameRes = -3,
+        kGfxMode_None    = 1,
+        kGfxMode_Desktop = 2,
+        kGfxMode_GameRes = 3,
```

Comment:
This will break the logic, as iirc these enums are used in a mix with array indexes.
It may be better to cast the type on assignment.
Alternatively, make these constants very large values, and possibly adjust the logic of their usage further in code.

Reply:
Right, the cast was what I was not able to do with the minus value, the compiler was giving me an error, I think the high bits strategy may be easier, so I will possibly may be going that way! Thanks fo catching it!

Reply:
I think I made a confusion, moving the cast to the switch clause does fix!